### ISSUE TITLE
Why does our default testmod set use_ssre = .true.?

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/CTSM#1186]
Why does the clm-default testmod set use_ssre = .true.? It seems like this is something that should be covered in maybe a single test, and that our default testmod should be covering something closer to the standard out-of-the-box configuration.

Maybe it was done this way to avoid creating a separate test just to cover this small option. I fully support trying not to add a test just to cover one small thing. But we could have a test that includes this along with a number of other options that make some other changes from the default configuration.

**Comment by ekluzek** [comment]
Happened to see this again, so putting a comment. The answer to the question is undoubtably the latter, turning something on that  could easily always be on, to prevent having to do a seperate test for it.

It's also only turning on diagnostic fields so a good thing to check in multiple configurations. Having just one test for it might not be sufficient. It can't always be on, so there are tests that explicitly turn it off.

My feeling is to leave it as is. But, happy to discuss. So adding next to it.

**Comment by samsrabin** [comment]
Makes sense, but it would be good to have a comment in the `user_nl_clm` file to explain this.
